common_utils.py

```diff
@@ -365,9 +368,6 @@ def import_annotations_dialog_clicked(self, button):
         BUTTON_ROLES = ['AcceptRole', 'RejectRole', 'DestructiveRole', 'ActionRole',
                         'HelpRole', 'YesRole', 'NoRole', 'ApplyRole', 'ResetRole']
         if self.dialogButtonBox.buttonRole(button) == QDialogButtonBox.AcceptRole:
-            # Remove initial_dialog_text if user clicks OK without dropping file
-            if self.text() == self.rac.initial_dialog_text:
-                self.pte.clear()
```

Comment:
Moon Reader tries to make a guess at the file path, since it always picks `<date>.mrpro` -- with this code, it doesn't let a user accept the default value.

Reply:
What are the values for these function result and the property:

* `self.text()`
* `self.rac.initial_dialog_text` 

Not sure what removed code actually does for other reader classes. We surely need to understand that before removing it.

Reply:
self.text() contains the contents of the exporting reader dialog window.  The initial dialog tests for the other exporting readers are already blank:

https://github.com/davidfor/calibre-annotations/blob/8f0104a35f2919c1780e0758266159febc891e21/readers/_BluefireReader.py#L88

https://github.com/davidfor/calibre-annotations/blob/8f0104a35f2919c1780e0758266159febc891e21/readers/GoodReader.py#L82

Except for Marvin, which just tells the user to drag and drop a file.

Clearing the field if the text didn't change doesn't really make much sense to me, and I don't think it breaks anyone.

Reply:
I see. Still better to do this in separate PR, since it's not breaking Moon Reader integration.